modules/analysis/rosbag-image-pipeline/images/ros-to-png/src/requirements.txt

```diff
@@ -0,0 +1,5 @@
+boto3
```

Comment:
Did you generate these requirements using pip-compile requirements.in?
As a standard, we declare the requirements in requirements.in and use pip compile to generate requirements.txt?